### PR TITLE
Catch errors during production of kernel dill

### PR DESCRIPTION
**Pull request by jonahwilliams**
~~This can't land until the sdk roll with the fix lands as well.~~

The fix has rolled though, and this will output kernel compilation errors in the future

### CODE REVIEW

File: web_sdk/flutter_kernel_sdk.dart [hunk 1]
```diff
@@ -2,16 +2,24 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+ // ignore_for_file: uri_does_not_exist
```
Comment (zanderso) [inline web_sdk/flutter_kernel_sdk.dart]: Why don't the import uris exist?

Comment (jonahwilliams) [reply web_sdk/flutter_kernel_sdk.dart]: They do, provided you use the .packages file from third part, but locally the analyzer doesn't find it and that blows up on this file

Comment (jonahwilliams) [reply web_sdk/flutter_kernel_sdk.dart]: * third_party

Comment (zanderso) [reply web_sdk/flutter_kernel_sdk.dart]: Could you add that in a comment here? Thanks!

Comment (jonahwilliams) [reply web_sdk/flutter_kernel_sdk.dart]: Done

**Pull request merged**